main.sh

```diff
@@ -252,20 +280,21 @@ for test in "$@"; do
 	    echo "DAOS_TEST is temporarily disabled - TBD"
             ;;
         SELF_TEST)
-            #start_server
-            #run_cart_test
-            #killall_proc $test
-	    echo "SELF_TEST is temporarily disabled - DAOS-3838"
-            ;;
-        MDTEST)
             start_server
-            start_agent
-            create_pool
-            run_mdtest
+	    dump_attach_info
+            run_cart_test
             killall_proc $test
+            ;;
+        MDTEST)
+            #start_server
```

Comment:
Same as above.

Reply:
I have not updated the dmg cmd to work with daos master. So the create pool is expected to fail.
I want to give Aruna the opportunity to update and enable IOR and MDTEST in her PR.